### ISSUE TITLE
An error when adding private `attribute` to an aliase

### ISSUE THREAD

**Issue by DeeGeeMan** [casey/just#1467]
Using the example from the document:
```
[private]
foo:

[private]
alias b := bar

bar:
```
I get the following error in just v1.9.0
```
error: Expected '*', ':', '$', '=', identifier, or '+', but found ':='
    |
385 | alias b := bar
    |         ^^

```

**Comment by DeeGeeMan** [comment]
I don't get an error when assigning an attribute to a recipe only.

**Comment by neunenak** [comment]
My PR that added support for adding a `[private]` attribute to an alias ( https://github.com/casey/just/pull/1434 ) was merged after version 1.9.0 was released. If you use a version of just compiled from the latest master branch, your justfile will work. @casey maybe it'd be a good idea to release a 1.10 version to get that change into an officially-released version of just.

**Comment by neunenak** [comment]
Also I think this is confusing because the documentation on just.systems gets updated as soon as a pull request merges, even though the changes in that pull request won't be available to people who aren't compiling just from latest master themselves until a new just release gets cut. I just checked https://just.systems/man/en/chapter_48.html and I noticed that the line that my PR changed now has a superscript "master" - I'm not sure how that gets generated and I don't think it's obvious that this means "this line of documentation only applies to the master branch".

**Comment by casey** [comment]
Sorry for the confusion! Just published 1.10.0 that contains this change. I opened #1469 to discuss making the docs less confusing regarding what's in releases vs what's in master.
